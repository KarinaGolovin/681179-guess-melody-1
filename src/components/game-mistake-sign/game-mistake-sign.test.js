import React from 'react';
import renderer from 'react-test-renderer';
import GameMistakeSign from './game-mistake-sign.jsx';

const mock = {
  mistakeSignState: `correct`,
};

it(`GameMistakeSign correctly rendered`, () => {
  const component = renderer.create(
      <GameMistakeSign
        mistakeSignState={mock.mistakeSignState}
      />
  ).toJSON();

  expect(component).toMatchSnapshot();
});
