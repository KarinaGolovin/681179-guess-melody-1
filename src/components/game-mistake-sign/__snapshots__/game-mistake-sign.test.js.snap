// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameMistakeSign correctly rendered 1`] = `
<div
  className="correct"
/>
`;
